packages/react-scripts/template/README.md

```diff
@@ -406,7 +406,7 @@ Then in `package.json`, add the following lines to `scripts`:
 ```diff
    "scripts": {
 +    "build-css": "node-sass src/ -o src/",
-+    "watch-css": "npm run build-css && node-sass src/ -o src/ --watch --recursive",
++    "watch-css": "npm run build-css && node-sass src/*.scss -o src/ --watch --recursive",
```

Comment:
ok, updating. I coudn't find anything at first in the readme of `node-sass` but looking through the code of the cli, yeah normal globbing is supported. updating.  `.sass` files are supported, but I think for a readme this should suffice no?